xcodeproj/internal/opts.bzl

```diff
@@ -501,19 +537,59 @@ under {}""".format(opt, package_bin_dir))
 
     return unhandled_opts
 
+def _process_user_swiftcopts(opts):
+    """Processes user-provided Swift compiler options.
+
+    Args:
+        opts: A `list` of Swift compiler options.
+
+    Returns:
+        A `list` of search paths.
+    """
+
+    quote_includes = []
+    includes = []
+
+    def process(opt, previous_opt):
+        if previous_opt == "-Xcc" and opt.startswith("-iquote"):
```

Comment:
We still need to handle `-Xcc -I -Xcc path`, right? I think a TODO is fine for now though.